src/resources/postcss/components/skeleton/_loader.pcss

```diff
@@ -21,15 +21,17 @@
 			padding-top: calc(var(--spacer-13) * 3);
 		}
 	}
-
+	
 	.tribe-common-c-loader__dot {
-		background-color: var(--color-accent-primary-background);
 		height: 15px;
```

Comment:
let's remove this

Reply:
it's gonna be removed already, am I missing something? or maybe you meant the height